### PR TITLE
Add host description in a couple of screens

### PR DESCRIPTION
**Pull request by nixocio**
Add host description in a couple of screens

See: https://github.com/ansible/awx/issues/3348
Also: https://github.com/ansible/awx/issues/9363

image

image

### CODE REVIEW

File: awx/ui/src/components/Lookup/HostFilterLookup.js [hunk 1]
```diff
@@ -419,6 +419,7 @@ function HostFilterLookup({
             headerRow={
               <HeaderRow qsConfig={QS_CONFIG} isSelectable={false}>
                 <HeaderCell sortKey="name">{t`Name`}</HeaderCell>
+                <HeaderCell sortKey="description">{t`Description`}</HeaderCell>
```
Comment (AlexSCorey) [inline awx/ui/src/components/Lookup/HostFilterLookup.js:422]: When a user sorts by description does it just sort alphabetically by the first letter in the description?

**Review by AlexSCorey — APPROVED** [review]
This looks good to me.  Awesome job updating/adding tests.

File: awx/ui/src/components/Lookup/HostFilterLookup.js (hunk 1 above)
Comment (nixocio) [reply awx/ui/src/components/Lookup/HostFilterLookup.js:422]: Exactly

**Review by InnocentK — APPROVED** [review]
Looks good!

**Pull request merged**